idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -581,4 +581,17 @@ service WorkflowService {
       5: shared.ServiceBusyError serviceBusyError,
       6: shared.EntityNotExistsError entityNotExistError,
     )
+
+   /**
+   * ReapplyEvents applies stale events to the current workflow and current run
+   **/
+  shared.ListTaskListPartitionsResponse ListTaskListPartitions(1: shared.ListTaskListPartitionsRequest request)
+    throws (
+      1: shared.BadRequestError badRequestError,
+      3: shared.EntityNotExistsError entityNotExistError,
+      4: shared.LimitExceededError limitExceededError,
+      5: shared.ServiceBusyError serviceBusyError,
+      6: shared.ClientVersionNotSupportedError clientVersionNotSupportedError,
```

Comment:
is this error expected from this API ? If not, just remove it

Reply:
Yep, This API is not for the client. So, removed this error.